hazelcast-client/src/main/java/com/hazelcast/client/util/ClientAddressCancellableDelegatingFuture.java

```diff
@@ -51,7 +51,7 @@ public ClientAddressCancellableDelegatingFuture(ClientInvocationFuture future,
 
     @Override
     public boolean cancel(boolean mayInterruptIfRunning) {
```

Comment:
It is sending a request to remote to cancel not started executions.
Is there a way to achieve this with underlying future ?